cit3.0-web/src/components/UserListItem/UserListItem.js

```diff
@@ -0,0 +1,51 @@
+import { Row, Col, Button } from "react-bootstrap";
+import PropTypes from "prop-types";
+import { formatDate } from "../../helpers/helpers";
+
+const UserListItem = ({ user, handelEditAction, handleDeleteAction }) => (
+  <div key={user.id} className="user-table-row w-100 p-2">
+    <Row>
+      <Col>{user.name}</Col>
+      <Col>{user.email}</Col>
+      <Col>
+        {user.municipalities.length
+          ? user.municipalities.map((muni, index) => (
+              <p key={index}>{muni.name}</p>
+            ))
```

Comment:
Do the municipalities have a unique ID we can use here for the key instead of the index? Indices aren't the best idea for keys if we can help it since the order of the items can change

Reply:
There can be a collision, there should just be a prefix to index